ReactNativeClient/android/settings.gradle

```diff
@@ -1,4 +1,6 @@
 rootProject.name = 'Joplin'
+include ':@react-native-community_geolocation'
+project(':@react-native-community_geolocation').projectDir = new File(rootProject.projectDir, '../node_modules/@react-native-community/geolocation/android')
```

Comment:
Same here, that should not be needed thanks to auto-linking.

Reply:
Removed